pkg/reconciler/kubernetes/tektoninstallerset/install.go

```diff
@@ -262,6 +351,91 @@ func (i *installer) updateDeployment(existing *unstructured.Unstructured, existi
 	return err
 }
 
+func (i *installer) ensureStatefulSet(expected *unstructured.Unstructured) error {
+	i.logger.Debugw("verifying a statefulset",
+		"name", expected.GetName(),
+		"namespace", expected.GetNamespace(),
+	)
+
+	// update proxy settings
+	err := common.ApplyProxySettings(expected)
```

Comment:
inside `ApplyProxySettings` we have a check. that will ignore `StatefulSet`,
```
if u.GetKind() != "Deployment" {
```

Do we really need proxy things to StatefulSets?

Reply:
I am not sure do we really need it but added considering what if require by someone in disconnected cluster and all

But ya i need to add condition for statefulset